### PR TITLE
Update base image and add dvc to new ml-dev image

### PR DESCRIPTION
**Pull request by anishpdalal**
# Overview

Frequently in ML projects, we find ourselves iterating on datasets and features rapidly. Easily share versions of new datasets, features, and models require a tool built for data versioning. This PR adds open-source tool DVC to orbyter so developers can more easily adopt data versioning workflows to a project from day 1. This PR also updates the python packages. Updating jupyter lab required to 3.0.0 requires node 12 so a new base image has also been added.

# Changes

* Added node.js v12 to base-sys image
* Added DVC to ml-dev image
* Update existing python packages in ml-dev

### CODE REVIEW

**Comment by mstefferson** [comment]
Can you add `mypy==0.790` to this? https://github.com/python/mypy

File: orbyter-ml-dev/3.2/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+isort==5.7.0
+jsonlines==1.2.0
+jupyter==1.0.0
+jupyter-contrib-nbextensions==0.5.1
+jupyterlab==3.0.0
+lightgbm==3.1.1
+matplotlib==3.3.3
+mlflow==1.13.1
+more-itertools==8.6.0
+notebook==6.1.6
```
Comment (mstefferson) [inline orbyter-ml-dev/3.2/requirements.txt:20]: `mypy==0.790`

**Review by resdntalien — APPROVED** [review]
lgtm, add the mypy and this looks good to go! There's probably some stuff we want to add to cookiecutter too no? make commands and some stuff in the README?

**Comment by anishpdalal** [comment]
> lgtm, add the mypy and this looks good to go! There's probably some stuff we want to add to cookiecutter too no? make commands and some stuff in the README?

Yeah we'll want to automatically set up the .dvc repo through cookiecutter

**Pull request merged**